### DISCUSSION TITLE
Welcome to MapartCraft Discussions!

### DISCUSSION THREAD

**Discussion by rebane2001** [rebane2001/mapartcraft#129] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 24056** [comment]
Когда выйдет на новую версию "на 1.19.4"?
